modules/ROOT/pages/diagram-tasks.adoc

```diff
@@ -111,9 +111,9 @@ To create a new task, select the relevant icon from the *BPMN elements* menu or
 menu of the preceding element in the diagram, and drop it on the whiteboard. You can change the task type
 of an task in the diagram.
 
-== Setting a unique task name in the Portal
+== Setting a unique task name in the Applications
```

Comment:
Setting a unique task name in Bonita Applications. 

Portal = Bonita Applications.

Reply:
I added it